components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -377,6 +377,8 @@
 <!-- bottom_drag -->
 <config_use_implicit_bottom_drag>.true.</config_use_implicit_bottom_drag>
 <config_implicit_bottom_drag_coeff>1.0e-3</config_implicit_bottom_drag_coeff>
+<config_use_implicit_top_drag>.false.</config_use_implicit_top_drag>
+<config_implicit_top_drag_coeff>1.0e-3</config_implicit_top_drag_coeff>
```

Comment:
@cbegeman, I think this value needs to be the same as for explicit top drag above:
```
<config_implicit_top_drag_coeff>2.5e-3</config_implicit_top_drag_coeff>
<config_implicit_top_drag_coeff ocn_grid="oEC60to30v3wLI">4.48e-3</config_implicit_top_drag_coeff>
<config_implicit_top_drag_coeff ocn_grid="ECwISC30to60E1r2">4.48e-3</config_implicit_top_drag_coeff>
<config_implicit_top_drag_coeff ocn_grid="SOwISC12to60E2r4">4.48e-3</config_implicit_top_drag_coeff>
<config_implicit_top_drag_coeff ocn_grid="ECwISC30to60E2r1">4.48e-3</config_implicit_top_drag_coeff>
```

Reply:
Very sorry I missed this!